__tests__/endpoints/generic.test.js

```diff
@@ -0,0 +1,15 @@
+const { app, data, seed, db, request } = require("../../testImports");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("/* - will return an error 400 with a suitable message when an incorrect endpoint is used", () => {
+	test("GET: 400 Returns an error if incorrect path is used", () => {
+		return request(app)
+			.get("/foobar")
+			.expect(400)
```

Comment:
404 makes more sense for this as it's not found - not a syntax error (as long as path is a string)